.github/workflows/smoke-checks.yml

```diff
@@ -1,19 +1,10 @@
 name: Smoke Checks
 
 on:
-  push:
-    branches:
-      - main
-      - develop
-
```

Comment:
Why are we removing this?

Reply:
Discussed this with @nuno-vieira, and we decided to remove it eventually.